js/plugin.iiif-iabookreader_strawberry.js

```diff
@@ -288,27 +330,80 @@ BookReader.prototype.loadManifest = function () {
 BookReader.prototype.setupTooltips = function() {
 };
 
+function processBehaviorToIABookreaderModeV3(jsonLd) {
```

Comment:
Tested with
```
"behavior":["paged"],
```
and
```
"behavior":["individuals"],
```
In a IIIF v3.0 manifest, and I can confirm this is working.